packages/x-tailwindcss/src/x-tailwind-plugin/components/tag/default.ts

```diff
@@ -0,0 +1,59 @@
+import { TailwindHelpers } from '../../../types';
+import { tagSizes } from './sizes';
+
+/**
+ * Returns the default styles for component `tag`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function tagDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignContent: 'center',
+    flexFlow: 'row nowrap',
+    boxSizing: 'border-box',
+
+    backgroundColor: theme('colors.neutral.0'),
+    borderColor: theme('colors.neutral.25'),
+    borderStyle: 'solid',
+    borderWidth: theme('borderWidth.1'),
+    color: theme('colors.neutral.75'),
+
+    gap: theme('spacing.8'),
+
+    fontFamily: theme('fontFamily.main'),
+    fontWeight: theme('fontWeight.regular'),
+    letterSpacing: theme('letterSpacing.lg'),
+    lineHeight: theme('lineHeight.sm'),
+
+    cursor: 'pointer',
+
+    '&:hover,&:focus,&:active': {
+      borderColor: theme('colors.neutral.50'),
+      color: theme('colors.neutral.90')
+    },
+
+    '&:disabled': {
+      cursor: 'not-allowed',
+      borderColor: theme('colors.neutral.25'),
+      color: theme('colors.neutral.25')
+    },
+
+    ...tagSizes(helpers).md,
+
+    '&:not(:disabled).selected': {
```

Comment:
I am not sure about this approach.
If we want to modify the color of the selected variant to be `colors.neutral.75` for example, we would need to add an `.tag.selected: color !important` or use the `.tag:not(:disabled).selected` selector in the Tailwind's component config, which seems a bit complicated.
@tiborux what do you think?

Reply:
You're right, not easy to modify the color. I've changed it with the refactor you suggested. Ty!